### DISCUSSION TITLE
Baudrate setting

### DISCUSSION THREAD

**Discussion by RahulMvr2824** [kenjdavidson/react-native-bluetooth-classic#270] [Q&A]
Hi there,
Do we need to set any baudrate in this library for smooth data transfer. If yes how do we set the baudrate.

**Comment by kenjdavidson** [comment]
A quick google shows that there is no way to set the baud rate for SPP connections.  It looks like the only way to do this is to connect using a different profile(s), which this library doesn't support.  So in order to do this you'll need to write your own custom Bluetooth Profile library or find one.
